### DISCUSSION TITLE
Move wandb project and experiment names to command line arguments?

### DISCUSSION THREAD

**Discussion by ungi** [SlicerIGT/UsAnnotationExport#42] [Ideas]
@chriscyyeung do you think we should move wandb_project_name and experiment_name from the config file to command line arguments? We need to share config yaml files as we add new options to our shared train script. But we will be continuously overwriting each other's project names.

**Comment by chriscyyeung** [comment]
That makes sense. It's definitely something that would change all the time. I can make a commit later today to add this in.

**Reply by ungi** [reply]
Thanks!
